cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -415,6 +414,11 @@ func (cmd *command) printSummary(o deployment.Overrides) error {
 }
 
 func (cmd *command) importCertificate() error {
+	if !cmd.approveImportCertificate() {
+		//no approval given: stop import
+		return nil
```

Comment:
Please print manual import instructions here. The user might want to import the certificate himself later.
Just need to call `cert.Instructions()`

Reply:
aaah .. good point. Will add that.